frontend/app/types/card.ts

```diff
@@ -2,7 +2,11 @@ export interface CardData {
   id: number;
   title: string;
   content: string;
```

Comment:
we should not have the title and content anymore and ID should be a string now. Was typescript not complaining about this?